src/ui/components/layouts/base_layout/side_nav.rs

```diff
@@ -0,0 +1,60 @@
+use crate::{
+  ui::components::common::icon::{Icon, IconType},
+  use_i18n,
+};
+use leptos::*;
+use leptos_i18n::t;
+use leptos_router::A;
+
+#[component]
+pub fn SideNav() -> impl IntoView {
+  let i18n = use_i18n();
+
+  view! {
+    <nav aria-label="Pages nav">
+      <ul>
+        <NavLink href="/create_post" icon=IconType::CreatePost text=t!(i18n, create_post)/>
+        <NavLink
+          href="/create_community"
+          icon=IconType::CreateCommunity
+          text=t!(i18n, create_community)
+        />
+        <NavLink href="/communities" icon=IconType::Communities text=t!(i18n, communities)/>
+        <NavLink href="/search" icon=IconType::Search text=t!(i18n, search)/>
+        <NavLink href="/modlog" icon=IconType::Modlog text=t!(i18n, modlog)/>
+        <NavLink href="/instances" icon=IconType::Instances text=t!(i18n, instances)/>
+        <NavLink href="/legal" icon=IconType::Legal text=t!(i18n, legal)/>
+      </ul>
+    </nav>
+    <nav aria-labelledby="lemmy-resources-label" class="mt-auto">
+      <div id="lemmy-resources-label" class="font-medium mb-1">
+        Lemmy Resources
+      </div>
+      <ul>
+        <NavLink
+          href="https://join-lemmy.org/docs/en/index.html"
+          icon=IconType::Documentation
+          text=t!(i18n, docs)
+        />
+        <NavLink href="https://github.com/LemmyNet" icon=IconType::Code text=t!(i18n, code)/>
+        <NavLink href="https://join-lemmy.org/" icon=IconType::Info text=t!(i18n, about)/>
+        <NavLink href="https://join-lemmy.org/donate" icon=IconType::Donate text=t!(i18n, donate)/>
+      </ul>
+    </nav>
+  }
+}
+
+#[component]
+fn NavLink(href: &'static str, icon: IconType, #[prop(into)] text: TextProp) -> impl IntoView {
+  view! {
+    <li>
+      <A
+        href=href
+        class="text-sm block leading-relaxed hover:bg-base-100 p-1.5 rounded-md transition duration-500 my-1 aria-current-page:bg-secondary aria-current-page:text-neutral aria-current-page:hover:bg-secondary/[0.6]"
```

Comment:
I'm not sure what some of these classes are for, but the responsive nav links in a drawer sidebar appear to be here: https://daisyui.com/components/drawer/#responsive

Reply:
That makes sense if we end up using all the same links in the hamburger menu as the side bar. I don't think they'll end up exactly the same though since the bottom nav makes several of them redundant on mobile.